### ISSUE TITLE
Migrate to a new host

### ISSUE THREAD

**Issue by camdendotlol** [camdendotlol/topstersorg#39]
Vercel has announced a new pricing model that affects Topsters 3's ability to remain in the free Hobby tier: https://vercel.com/blog/improved-infrastructure-pricing

We've been falling well within the bandwidth limits, even when the site went viral in September 2023, but they're adding limits on total requests (i.e. not just page views but each resource loaded: images, JS files, etc.) that will be impossible to avoid exceeding.

There seems to be a six month grace period so I have some time to investigate other options. GitHub Pages seems like the simplest choice at the moment. Netlify might be more straightforward.

**Comment by camdendotlol** [comment]
Well I tried Netlify and it didn't go well. They generated an HTTPS cert for the `*.netlify.app` domain when I created the app. When I added `topsters.org` to the app, Netlify continued to serve the cert for the `*.netlify.app` domain, causing a `bad cert domain` error that's really scary and many browsers won't allow users to bypass it. Normally I'd expect the host to just serve the non-HTTPS version of the site until the cert is ready, but I guess Netlify would rather sites be broken and "secure" than functioning.

I gave them a solid hour to update the certificate but they only displayed a generic error (same as [this user](https://answers.netlify.com/t/support-guide-troubleshooting-ssl-certificate-errors/39865/70)). I wonder if something was temporarily wrong with Netlify's systems. But being effectively down for over an hour is not cool and I think I'll look into GH Pages or something else instead.
